src/qt/rpcconsole.cpp

```diff
@@ -301,6 +302,14 @@ bool RPCConsole::RPCParseCommandLine(std::string &strResult, const std::string &
                             JSONRPCRequest req;
                             req.params = RPCConvertValues(stack.back()[0], std::vector<std::string>(stack.back().begin() + 1, stack.back().end()));
                             req.strMethod = stack.back()[0];
+#ifdef ENABLE_WALLET
+                            // TODO: Move this logic to WalletModel
+                            if (vpwallets.size() > 1)
+                            {
```

Comment:
Wouldn't this be safe to do for vpwallets.size()==1 too? If so, I'd slightly prefer that, because it means this code is always used when wallet is enabled.